app/front/scripts/controllers/provide-metadata.js

```diff
@@ -25,36 +32,41 @@ angular.module('Application')
           end: fiscalPeriod ? fiscalPeriod.end : ''
         };
 
-        $scope.$watch('attributes.title', function() {
```

Comment:
Yes, definitely - it could. But in this case there are no too much data - `attributes` object is almost plain and has no nested objects, so it's quite fast.